build.gradle

```diff
@@ -58,6 +60,36 @@ task cleanNode( type: Delete ) {
     delete 'node_modules'
 }
 
+// Command to find if any new files types should be excluded
```

Comment:
I used webpack copy plugin https://github.com/enonic/app-standardidprovider/pull/849/files
And gradle config got much simpler
https://github.com/enonic/app-standardidprovider/pull/849/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7L62